rational/test/test_trigo_rational.mligo

```diff
@@ -13,7 +13,7 @@ let test =
         let angle = Trigo.zero in
         let expected = (Rational.new 0) in
         let diff = Rational.sub (Trigo.sinus(angle, precision)) expected in
-        let error = Rational.resolve diff 12n in
+        let _error = Rational.resolve diff 12n in
```

Comment:
This line of code seems to be useless (same in the order tests)

Reply:
Done